types/bun/ws.d.ts

```diff
@@ -0,0 +1,370 @@
+/**
```

Comment:
This should be under `"ws"` instead of a global module

bun doesn't implement most of these types yet. it should just be `WebSocket` from lib.dom.d.ts